examples/egress/egress-policy.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "projectcalico.org/v1"
+kind: Egresspolicy
```

Comment:
Actually, I think that the schema defined in K8S API is not enough for us. It's pretty restrictive (as described in the PR 458), and one of our main problems also is using CIDR instead of Kubernetes Objects to define Network Policies. So I Agree using the policy from Calico would be great!

I'm only concerned about, as Calico is the main example for Kubernetes Community about Network Policies, how would this impact (positive or negative) in the project Image, as we're doing something 'outside the box'.

Maybe we could use both the approaches, and make something like 'extending Kubernetes Network Policies' while they don't officially support Egress (and other needs described in https://github.com/projectcalico/calico/issues/458#issue-202650311)